openid-4-verifiable-presentations-1_0.md

```diff
@@ -215,6 +215,53 @@ Figure: Cross Device Flow
+
+Note: The diagram does not illustrate all the optional features of this specification.
+
+Note: The usage of the Request URI as defined in [@!RFC9101] does not depend on any other choices made in the protocol extensibility points, i.e., it can be used in the Same Device Flow, too.
+
+!---
+~~~ ascii-art
+~~~ ascii-art
++--------------+   +--------------+                                    +--------------+
+|     User     |   |   Verifier   |                                    |    Wallet    |
++--------------+   +--------------+                                    +--------------+  
+        |                 |                                                   |
+        |    Interacts    |                                                   |
+        |---------------->|                                                   |
+        |                 |  (1) Discovery Request                            |
+        |                 |  (Presentation Request URI)                       |
+        |                 |-------------------------------------------------->|
+        |                 |                                                   |
+        |                 |  (2) Request the Request Object                   |
+        |                 |<--------------------------------------------------|
+        |                 |                                                   |
+        |                 |  (2.5) Respond with the Request Object            |
+        |                 |      (Presentation Definition)                    |
+        |                 |-------------------------------------------------->|
+        |                 |                                                   |
+        |   User Authentication / Consent                                     |
+        |                 |                                                   |
+        |                 |  (3)   Authorization Response                     |
+        |                 |  (VP Token with Verifiable Presentation(s))       |
+        |                 |<--------------------------------------------------|
+~~~
+!---
+Figure: Cross Device Flow
+
+(1) The Verifier sends to the Wallet a Discovery Request that contains a Presentation URI from where to obtain the Request Object containing the actual Authorization Request object. 
+
+(2) The Wallet sends an HTTPS POST request to the Presentation URI to retrieve the Request Object.
```

Comment:
```suggestion
(2) The Wallet sends an HTTPS POST request to the Presentation URI that can contain information about wallet's endpoint, capabilities and attestation.
```

Reply:
wallet's endpoint -> Wallet's endpoint

Reply:
I adopted the text to convey what I perceive was the objective of the proposal, I used different text since there is no capabilities nor endpoint directly be conveyed in the POST request as it is specified now.